### DISCUSSION TITLE
[SC]: Sostituzione allegato già caricato

### DISCUSSION THREAD

**Discussion by migmar68** [italia/ansc#926] [Supporto servizi cooperativi] [answered]
### Ambiente

pre-produzione

### Servizio

R001 - Servizi cooperativi allegati

### idOperazione

assente

### Provenienza segnalazione

Software House

### Comune di riferimento e operatore

Poggio a Caiano

### Descrivi il problema

Buongiorno. Non abbiamo capito come sostituire un allegato caricato erroneamente, su un atto già presente in ANSC, ma ancora non firmato dallo USC. Sulla webapp è possibile, quale funzione occorre utilizzare nei web service?
più che sostituzione, vedendo la webapp è corretto parlare di chiamata per cancellazione, seguita da successiva chiamata per caricare di nuovo il documento.

**Comment by gbuemi** [comment, marked as answer]
Buon pomeriggio,
non è previsto un servizio specifico per la cancellazione o sostituzione dell'allegato. 

Può essere invocato però il servizio R009 /validazione/evento/{version} trasmettendo l'id dell'evento. Così facendo il servizio di validazione andrà in aggiornamento e se non si trasmettono i dati dell'eventuale allegato caricato in precedenza si raggiunge lo scopo.

Al momento stiamo lavorando sulla [issue 912](https://github.com/italia/ansc/issues/912) che evidenzia un problema proprio nello scenario appena descritto. Nel momento in cui verrà risolto il bug potrete effettuare un test.

Un saluto e grazie,
Supporto ANSC
